### PR TITLE
{devel}[dummy/dummy] Maven v3.6.0

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by sassy-crick** [comment]
Test report by @sassy-crick
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login002.camp.thecrick.org - Linux RHEL 7.3, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c2453df2aacce9f899481d26d513c101 for a full test report.

File: easybuild/easyconfigs/m/Maven/Maven-3.6.0.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+homepage = 'http://maven.apache.org/index.html'
+description = """Binary maven install, Apache Maven is a software project management and comprehension tool. Based on
+the concept of a project object model (POM), Maven can manage a project's build, reporting and documentation from a
+central piece of information.
+"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['apache-maven-%(version)s-bin.tar.gz']
+source_urls = ['http://apache.org/dist/maven/maven-%(version_major)s/%(version)s/binaries/']
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/m/Maven/Maven-3.6.0.eb]: why the change to `apache.org`, isn't it safer to keep using `archive.apache.org`?

Comment (sassy-crick) [reply easybuild/easyconfigs/m/Maven/Maven-3.6.0.eb]: Good question. I took the Maven-3.5.2.eb file from EB version 3.8.0 as a template and simply changed the version number and checksum. 
I am happy to use the archive.apache.org URL if you think that is a better way of doing it.

Comment (boegel) [reply easybuild/easyconfigs/m/Maven/Maven-3.6.0.eb]: @sassy-crick Didn't you mention that you ran into missing source files for `Maven`? This may be why, old tarballs are probably (only) on `archive.apache.org`?

Comment (migueldiascosta) [reply easybuild/easyconfigs/m/Maven/Maven-3.6.0.eb]: @sassy-crick ah, sorry, you didn't remove `archive`

but in the past we've had to change to `archive.apache.org` to fix broken urls after they are no longer available in `apache.org` (e.g. https://apache.org/dist/maven/maven-3/3.3.3/binaries/ is broken, https://archive.apache.org/dist/maven/maven-3/3.3.3/binaries/ is not), so I think it would be better to always use `archive.apache.org`... @boegel?

Comment (sassy-crick) [reply easybuild/easyconfigs/m/Maven/Maven-3.6.0.eb]: @boegel Yes. The problem here was that I did not find that version when I was looking at the webpage. Following the URL mentioned in the EB build file, I raised the issue in the chat. I did not check the archive.apache.org at the time. 
@migueldiascosta given the info I have now I suggest to change it over to the archive.apache.org if @boegel is happy with it and wants me to do that.
How do we handle this given it was added to the milestone? Still learning GitHub. :-)

Comment (boegel) [reply easybuild/easyconfigs/m/Maven/Maven-3.6.0.eb]: @sassy-crick Changing to `archive.apache.org` is the right way forward here, you should be able to do that easily use `--update-pr`.

The milestole attached to this PR doesn't matter, you can just push in additional commits to this PR (either manually or via `--update-pr`).

Comment (sassy-crick) [reply easybuild/easyconfigs/m/Maven/Maven-3.6.0.eb]: @boegel done. I agree, this is the right way of doing it. Let me know if there are any other issues.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.7, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/2cf0f02eae176a3e06b8628d4514dad0 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @sassy-crick!
